mitmproxy/addons/dumper.py

```diff
@@ -154,46 +155,54 @@ def _echo_request_line(self, flow):
             url = flow.request.pretty_url
         else:
             url = flow.request.url
-        terminalWidthLimit = max(shutil.get_terminal_size()[0] - 25, 50)
-        if ctx.options.flow_detail < 1 and len(url) > terminalWidthLimit:
-            url = url[:terminalWidthLimit] + "…"
+
+        if ctx.options.flow_detail <= 1:
```

Comment:
Did this condition change from `<` to `<=`?

Reply:
Yes. `<` does not make any sense because flow_detail = 0 means quiet...